library/common/config/config.cc

```diff
@@ -484,7 +484,7 @@ stats_sinks: *stats_sinks
         - safe_regex:
             regex: '^pulse.*'
         - safe_regex:
-            regex: '^vhost.api.vcluster\.[\w]+?\.upstream_rq_(?:[12345]xx|retry.*|time|timeout|total)'
+            regex: '^vhost.*.vcluster\.[\w]+?\.upstream_rq_(?:[12345]xx|retry.*|time|timeout|total)'
```

Comment:
This will match `vhostXYZfoo.vcluster..." is that intended?

Should we match a period before the vhost name?

Maybe something like this? `^vhost\.\w+\.vcluster`

Reply:
Good points - fixed! Thank you!